pkg/crawler/colly.go

```diff
@@ -49,19 +47,29 @@ import (
 	"github.com/gocolly/colly/extensions"
 )
 
+type Results struct {
+	Results    []string
```

Comment:
I was thinking that maybe it's better to call this field `URLs`, just to avoid repetition `Results.Results`.
I feel `Results.URLs` is better